llvm/lib/Support/CMakeLists.txt

```diff
@@ -198,4 +198,4 @@ if(LLVM_WITH_Z3)
     )
 endif()
 
-target_link_libraries(LLVMSupport PRIVATE LLVMMLBridge)
```

Comment:
Is this change intentional? I think PUBLIC linking is not required in any of the places. Can you please confirm?

Reply:
In this case, it wouldn't be a problem changing it to _PRIVATE_, but it would be, if big changes are there in _LLVMSupport_ which depend on _MLBridge_(since _include_directories_ won't be propagated). 

Also, for dynamic passes, it can linked as  _PRIVATE_. I changed it to _PUBLIC_ to maintain consistency.